system/jlib/jstats.cpp

```diff
@@ -377,7 +377,13 @@ static StringBuffer & formatLoad(StringBuffer & out, unsigned __int64 value)
 static StringBuffer & formatSkew(StringBuffer & out, unsigned __int64 value)
 {
     //Skew stored as 10000 = perfect, display as percentage
-    return out.appendf("%.2f%%", ((double)(__int64)value) / 100.0);
+    const double percent = ((double)(__int64)value) / 100.0;
+    if (value > 1000)      // > 10% => display whole numbers only
```

Comment:
@jakesmith what do you think?
@shamser really picky, but are these the correct thresholds?  It means that you have at most 2 digits, and often only 1 and a bit significant digits.  E.g., 105 is displayed the same as 114.  I think that loses too much informatin, so I would have an extra digit, (and use >= rather than >).

Reply:
I'm not sure what you mean by 105 is displayed same as 114.  Value 105 will be displayed as 1.0% and 114 will be displayed as 1.1%... should we have higher precision for small numbers?

Reply:
105 will also be displayed as 1.1% (since it is floating point and will round up).  If it was truncated then 1.10% and 1.19% would be displayed the same.  Having an accuracy of only 10% for some numbers isn't really high enough.

Reply:
Sorry, my mistake.  Should we have 2 digits of precision for anything less than 10%?

Reply:
Yes, I think multiply all your test values by 10, but also make the test >= so that the number of digits stays consistent from 9.99% to 10.0%

Reply:
I'm not sure it matters much either way (I can't think where I've looked at graph/skew and cared about such a small skew, or that marginally different).

But don't have a problem with 2 decimal places consistently either. 
I think the original request was on silly large numbers 10000's% with decimals, which leads to not nice display issues.
So the main issue is that they get long with no benefit.
I suspect trying to limit to 5 digits in total is good enough, so >=10000% whole num, >=1000, to 1 decimal digit, < 1000 = 2 decimal digits.